travis/minimesosFile

```diff
@@ -2,7 +2,7 @@ minimesos {
     clusterName = "Cook Simulation Cluster"
     loggingLevel = "INFO"
     mapAgentSandboxVolume = true
-    mapPortsToHost = false
+    mapPortsToHost = true
```

Comment:
I wasn't sure how to find the IP for zookeeper otherwise. Also, it mirrors the dev set up so it is less likely to cause problems where in dev tests pass but in integration they don't